hedera-node/src/main/java/com/hedera/services/legacy/services/state/AwareProcessLogic.java

```diff
@@ -85,30 +86,31 @@ public AwareProcessLogic(ServicesContext ctx) {
 	}
 
 	@Override
-	public void incorporateConsensusTxn(Transaction platformTxn, Instant consensusTime, long submittingMember) {
-		try {
-			final var accessor = new PlatformTxnAccessor(platformTxn);
-			Instant effectiveConsensusTime = consensusTime;
-			if (accessor.canTriggerTxn()) {
-				effectiveConsensusTime = consensusTime.minusNanos(1);
-			}
+	public void incorporateConsensusTxn(Transaction platformTxn,
+										PlatformTxnAccessor accessor,
+										Instant consensusTime, long submittingMember) throws InvalidProtocolBufferException{
```

Comment:
nit: the indentation seems to be way off here

Reply:
Fixed